src/v2/Apps/Auctions/auctionsRoutes.tsx

```diff
@@ -11,6 +11,24 @@ const AuctionsApp = loadable(
     resolveComponent: component => component.AuctionsAppFragmentContainer,
   }
 )
+const AuctionsRoute = loadable(
+  () =>
+    import(
+      /* webpackChunkName: "auctionBundle" */ "./Routes/Auctions/AuctionsRoute"
+    ),
+  {
+    resolveComponent: component => component.AuctionsRouteFragmentContainer,
+  }
+)
+const ArtworksRoute = loadable(
+  () =>
+    import(
+      /* webpackChunkName: "auctionBundle" */ "./Routes/Auctions/AuctionsRoute"
```

Comment:
Just wondering if this should be `./Routes/Artworks/ArtworksRoute`?

Reply:
Ah, good eye, will update the variable name 👍